### DISCUSSION TITLE
Interpreting the alpha spectrometer output

### DISCUSSION THREAD

**Discussion by SteveF161** [ozel/DIY_particle_detector#29] [Q&A]
My son and I have built our alpha spectrometer and it works first time, with lots of nice pulses and a good detection rate. Source is a shard of Fiestaware pottery with a uranium glaze. I did a recording run with about 40,000 detected alpha pulses but the spectrum I get looks very different to what I expect. I wasn't expecting to get anything like the simulation plots, but neither was I expecting to  see a peak at the low energy end of the range - was expecting abroad peak around 4Mev(-ish). I've checked obvious things like recording level on my sound card. Does anyone have any ideas why I'm seeing what I'm seeing?
![Test Run almost 40k pulses](https://user-images.githubusercontent.com/28903476/103950738-540a2680-5135-11eb-9f3c-d4a7836d3369.png)

(Data file available but it's 318MB)

**Comment by ozel** [comment]
Nice progress, this measurement doesn't look so unexpected actually! 
From uranium glaze, you will get much more electrons (and gammas) than alphas. Note how in my plot below the left "electron edge" is cropped and the y-range is zoomed in for focusing on the higher alpha energy range (>1 MeV):
![](https://raw.githubusercontent.com/ozel/DIY_particle_detector/master/images/Alpha_spectrum_Majolika_ceramic.png)
The distance might be too large for properly registering alphas in your setup. I always put the metal case of the diode in direct contact with the ceramics, such that there are only 2-3mm of air left between the glaze and the silicon chip.
If you have bad luck with your sample, there could be a thick transparent layer of paint on top of the uranium glaze, absorbing all the alphas. But personally, I have not observed that possibility with about 5 different ceramics samples. 

You are using now the CM108 USB soundcard, right? If you have it at 100% and with gain boost *enabled* as [explained in the Wiki](https://github.com/ozel/DIY_particle_detector/wiki/Soundcards#operating-system-specific-tips), the coarse threshold for alphas is around -1250 ~= 500 keV (or about -8 mV on a [regular oscilloscope input](https://github.com/ozel/DIY_particle_detector/wiki/Oscilloscope-Measurements#alpha-spectrometer))
There is some grey area of course, but usually, about 1 MeV should be the absolute upper maximum of energy that an electron may deposit due to the constrained sensor volume (which can also vary a bit from diode to diode, cf. the paper :-).
That your electron peak goes further towards 2 MeV could be simply caused by pile-up (more than one pulse at the same time) because the sample is rather active which is often the case with uranium glaze. Try choosing another spot, more towards the edge of the uranium glazed area. Please post a picture of the whole setup if possible.

I would advise you to use the code snipped labelled [FULL ENERGY RANGE histogram](https://github.com/ozel/DIY_particle_detector/blob/53a7770a66fe0209b38a826d560bc8a4b6b56a0d/data_analysis_and_reference_measurements/diode_detector/analyse_and_plot_pulses.py#L448) instead of the reference measurement plot that you used.
You have to experiment with `max_y_limit` for every measurement, for yours above something like 100 or 50 could maybe reveal some structure. Also, check for the highest energy pulses (besides a few double pulses that may always pile up: alpha+alpha, or alpha+electron at the same time) it should be below 5 MeV as in my measurement. More air (or very dense air) and additional "passive" layers of paint between diode and sensor will lower the recorded energies and shift the alpha spectrum further to the left. In case you have put a capacitor foil on the diode (see WiKi "Diodes" page) that will subtract even more energy. I prefer to just take a large enough metal box where the specimen fits inside completely as opposed to shielding the diode from light with additional metal foils.

My reference alpha source, by the way, was an almost exclusive alpha emitter, besides few low-energy X-rays. The simulation software did not even simulate those, that's why the simulation is flatter in the very low energy regime than my [reference measurement in figure 4](https://www.mdpi.com/1424-8220/19/19/4264/htm).

**Comment by SteveF161** [comment]
Hi Oliver

The soundcard I'm using is the Sabrent AU-MMSA. This is also a very low-cost sound card (I paid around £5). I don't know if it is the same chipset as the one that you've used but I think it is highly likely that it is given my experience with low-cost manufacturers. It's plugged directly into a USB port on my Raspberry Pi4.

My alpha source is a shard of red Fiestaware pottery and I'm placing it directly on top of the photodiode casing with the glazed side facing directly to the chip. The shard is small and the box (...when the lid is on!...) is well earthed via the shielded BNC cable.  

I had THL set to -1000 during this collection run and same in the analysis run as I was hitting memory limits on the Pi4 with >100,000 pulses. I used this value so as to ignore most of the electron pulses.

I will reprocess the data but won't get a chance until Sunday as  I have my first university exam in 40 years tomorrow!

Steve

![IMG_1029](https://user-images.githubusercontent.com/28903476/104715522-9677c880-571e-11eb-9266-4ec7fab21b67.JPG)
![IMG_1028](https://user-images.githubusercontent.com/28903476/104715531-98418c00-571e-11eb-993b-de4ab8f6c39e.JPG)

**Reply by SteveF161** [reply]
I've now rerun the data plot. I still have THL =-1000 as before. In order to minimize the huge electron peak, I've also put a small change into the code so that it only plots pulses with peak >= min_alpha_peak. I also took the bin size to 67*4 just in case the data was spread too thin. I am clearly seeing activity over a wide energy range but I just cannot interpret it. (I don't think it is caused by noise. Visually, the peaks look to be perfect alpha peaks.)
![image](https://user-images.githubusercontent.com/28903476/104838269-c73c3700-58b1-11eb-941e-9f975866c24a.png)

I would try a different alpha source if I had one. Is uranium glass really a bad choice?

**Reply by ozel** [reply]
How long have you been running the measurements?
And what is the average count rate? If pulses are too close following each other, they would pile-up as one.
You can check for this by setting `SHOW_DETECTED_PULSES` and `OVERLAY_PULSES` to `True. 
It will create those overlay/persistence style plots like in the paper figure 9a). This will take quite some time but show you if there is a lot of noise or pile-up wrongly recorded as a single pulse.

Again, you need to zoom into the recorded high energy part of the plot by either reducing the y-limit or increasing the THL value a lot more (the first option is better for debugging but records more data obviously).

Your soundcard could very well have the CM108 USB chip inside (you could check the USB product and vendor IDs in the device manager or with `lsusb`) but I am surprised that your data goes well beyond 8 MeV. Please increase the x-limit of the plot as well to see where the spectrum actually ends. It's likely that you see pile-up and/or that your soundcard or system settings amplify the input more than my CM108. Another possibility is that you have some additional software gain enabled besides the soundcards own gain boost. Check your alsamixer settings if this is still under Linux/RPi.

If you don't have pile-up/vibration noise and your pulses look good in the overlay plot, I would suggest recording for 30-35 hours straight with a fresh battery or better charged 9.6V NiMH accumulator. Put some additional thick dark piece of clothing on top of the box to be sure that there is no light reaching inside - it will also protect it a bit better from acoustic vibrations and the microphonic effect.
You can also combine several recordings form several nights, each with a freshly charged accumulator. See the wiki on "Batteries".
A depleted/almost empty battery will distort the spectrum as well.

Your small ceramics sample should be perfect. Just be careful with the paint chips and prevent contamination. 
Pile-up is rather unlikely due to its small size but certain glazes are extremely active.

Make sure to not touch the tin bix, *table* or cable at all during any recordings - basically, do not use the table for anything else while recording. It is very sensitive to tiny vibrations, which you'd see as artefacts in the pulse overlay plots.

**Reply by SteveF161** [reply]
Hi Oliver, 

Strangely, I can't find any details of the chip in my sound card (Sabrent AU-MMSA). "lsusb" identifies it as a "Unitek Y-247A". Neither the Sabrent nor the Unitek data sheets show the chip set, which is very unhelpful! I'll write to the manufacturer(s) for details. The soundcard is correctly configured in alsamixer and there is no additional gain control capability on input on the Raspberry Pi. The two low-cost sound cards that you mention in the wiki are unavailable in the UK at the moment. 

I do understand that a different amplification factor is going to affect how the energies of the particles appear. If my card uses a different chipset then I would need to calibrate against a known source, but I have no access to any.

The data collection was over about 10 hours:  `detected pulses: 27015 in 595 minutes -> 45.413 CPM`. I did hit some problems with large numbers of pulses as the internal copy within the Python script blows all available memory when it tries to save the data. I can try another run (...but I'm currently out of batteries!)

I couldn't see any obvious evidence of pile-up :
![Pulses overlaid](https://user-images.githubusercontent.com/28903476/104925798-3049aa80-5997-11eb-9153-b188f66c14d2.png)

I've set max_y_limit to a lower value, and made the histogram bins bigger. I may be imagining that there is some slight shape in the profile:
![Alpha pulses 18Jan21](https://user-images.githubusercontent.com/28903476/104925911-5a9b6800-5997-11eb-8f9c-e40b9c91ca52.png)

**Comment by ozel** [comment]
What are the hex numbers next to the Unitek device name in lsusb? Mine shows:
```Bus 001 Device 018: ID 0d8c:013c C-Media Electronics, Inc. CM108 Audio Controller```
The hex pair encodes vendor and product ids, but those could be changed even if the same CM108 is inside.

Your pulse overlay looks excellent, congratulations! Only few pile-up "ghosts" on the right side (difficult to see in the compressed image, but single faint pulses can be seen to the right).
The 45 CPM count rate confirms that your sample is not too radioactive. I've observed problems with distorted alpha spectra from 150-200 CPM upwards due to pile-up. Also 27.000 total counts usually enough to see a spectrum structure!
My prediction: We're looking at the U-238 "peak" which is turned into a long plateau because the glaze is probably rather thick, so most of the alphas lose a lot of energy before escaping the glaze.
In order to see the second part in the spectrum, the U-234 contribution, you have to zoom out on the x-axis by increasing the x-limit of the plot by a lot. 
It should be a smaller "foot" on the very left, about a fifth in height of the U-238 plateau. 
BTW, it reveals itself already a bit in the intensity variation of your overlayed pulse plot...

I will explain why it looks so different from my ceramics plot above in the next post, once you showed the full x-range. :-)

**Reply by SteveF161** [reply]
It's a C-Media chip, I think, but cannot confirm that it's the 108. I saw from their website that they make several others.

![image](https://user-images.githubusercontent.com/28903476/104945026-fafd8680-59af-11eb-9f39-c7d01457898c.png)

The full-range alpha plot (max_bin_kev=16000) is here:
![Full range alpha plot](https://user-images.githubusercontent.com/28903476/104946277-af4bdc80-59b1-11eb-8d54-8bf219f92111.png)

The shape looks more like I expected to see but that only makes sense if the amplification is different for my sound card, doesn't it?

**Reply by SteveF161** [reply]
..and I've now had confirmation from the vendor that by soundcard is based on CM-108

**Reply by ozel** [reply]
> The shape looks more like I expected to see but that only makes sense if the amplification is different for my sound card, doesn't it?

Yes! The x-range is distorted/different because of the amplification factor diverging from my setup.

The shape of the spectrum is not pretty but that is unavoidable if the glaze is rather thick.
You might be able to improve it by using a fresh battery. If the battery runs out during the measurement the spectrum will be more smeared than in reality. Also try to make sure that humidity, pressure and temperature do not vary too much (air density should stay as constant as possible). Then you might be able to see a clearer edge towards the right end.
The reason for the intensity difference (U-234 vs. 238) compared to my spectrum on the top is due to actual differences in the glaze composition. I am currently writing another paper about this curiosity, stay tuned :-).

Yours is definitely a CM108-series chip, the VID PID pair appears in several forum posts and ham radio projects. There is actually a way for setting the gain boost and other volume parameters via USB in these C-Media chips, independent of the operating system driver support! (check out the datasheets link on the wiki)

We should start a dedicated discussion thread, on gathering data about different CM108 style soundcards. With a USB HID read/write call, we may be able to identify, read and set the gain defaults such that different cheap CM108 cards have all the same settings regardless of their factory defaults! A quick test using `hid-recorder` from  https://gitlab.freedesktop.org/libevdev/hid-tools looks promising. (install as root and run as root)

**Reply by ozel** [reply]
you could also try your own energy calibration by using a better alpha source like a small uranium mineral stone or a different piece of ceramics with a thinner uranium glaze, try to look for older items from the beginning of the 1900s if possible. 
The two polonium peaks will always stick out clearly on the high energy end with uranium minerals - see in my MDPI Sensors paper figure 9b).

Maybe even the 'radioactive party balloon' experiment would already provide enough statistics if you select a good spot on the balloon for the measurement after polonium collection (from radon progeny). That spectrum is pretty pure, the activity will be quite low but since it is an almost perfect 'surface source' it could just work. would be really interesting, I haven't tried that myself yet.

radioactive balloon experiment in short: 
- inflate party balloon
- close with a knot 
- charge well with hair or wool for 1-2 minutes (wait until you hear high-voltage discharge spikes loud and clear)
- stick the balloon to a (non-conductive) _indoor_ wall with a piece of tape on the knot
- let it collect for 15 minutes
- slowly deflate by carefully cutting a tiny hole into the balloon right next to the knot
paper reference: [DOI 10.1088/0031-9120/32/2/016](https://sci-hub.st/10.1088/0031-9120/32/2/016) 
and also in my MDPI Sensors paper figure 6 a), but there it was measured with a much larger area pixel detector, not using the small diode spectrometer.
